composites/Plugin/Shared/components/KeywordInput.js

```diff
@@ -1,73 +1,89 @@
 import PropTypes from "prop-types";
+import uniqueId from "lodash/uniqueId";
+import { __ } from "@wordpress/i18n";
 
+// Internal dependencies.
 import colors from "../../../../style-guide/colors.json";
 
-let KeywordField = styled.input`
-	margin-right: 0.5em;
-	border-color: ${ props => props.borderColor };
+const errorColor = colors.$color_red;
+
+const KeywordInputContainer = styled.div`
+	display: flex;
+	flex-direction: column;
+	margin: 1em 0;
 `;
 
-const ErrorText = styled.div`
+const KeywordFieldLabel = styled.label`
 	font-size: 1em;
-	color: ${ colors.$color_red };
-	margin: 1em 0;
-	min-height: 1.8em;
+	font-weight: bold;
+	margin-bottom: 0.5em;
 `;
 
-const ErrorMessage = "Are you trying to use multiple keywords? You should add them separately below.";
+const KeywordField = styled.input`
+	border-color: ${ colors.$color_grey_light };
+	border-style: solid;
+	border-width: 3px;
+	padding: 0.75em;
+	font-size: 1.1em;
+
+	&.hasError {
+		border-color: ${ errorColor };
+		outline-color: ${ errorColor };
+	}
+`;
+
+const ErrorText = styled.p`
+	color: ${ errorColor };
+	margin: 0.5em 0 0 0;
+	min-height: 1.8em;
+`;
 
 class KeywordInput extends React.Component {
 	/**
 	 * Constructs a KeywordInput component
 	 *
-	 * @param {Object}      props          The props for this input field component.
-	 * @param {String}    	props.id       The id of the KeywordInput.
-	 * @param {IconsButton} props.label    The label of the KeywordInput.
-	 * @param {boolean}     props.keyword  The initial keyword passed to the state.
+	 * @param {Object}      props           The props for this input field component.
+	 * @param {string}      props.id        The id of the KeywordInput.
+	 * @param {string}      props.label     The label of the KeywordInput.
+	 * @param {boolean}     props.showLabel Toggle between an actual label or an aria-label on the input.
+	 * @param {string}      props.keyword   The initial keyword passed to the state.
+	 * @param {function}    props.onChange  The function that is triggered when the keyword input field is changed.
 	 *
 	 * @returns {void}
 	 */
 	constructor( props ) {
 		super( props );
 
-		this.onChange = this.handleChange.bind( this );
-
-		this.state = {
-			showErrorMessage: false,
-			keyword: props.keyword,
-		};
+		this.handleChange = this.handleChange.bind( this );
+		this.displayErrorMessage = this.displayErrorMessage.bind( this );
 	}
 
 	/**
 	 * Checks the keyword input for comma-separated words
 	 *
-	 * @param {String} keywordText The text of the input
+	 * @param {string} keywordText The text of the input
 	 *
-	 * @returns {void}
+	 * @returns {boolean} Returns true if a comma was found.
 	 */
 	checkKeywordInput( keywordText ) {
-		let separatedWords = keywordText.split( "," );
-		if ( separatedWords.length > 1 ) {
-			this.setState( { showErrorMessage: true } );
-		} else {
-			this.setState( { showErrorMessage: false } );
-		}
+		return keywordText.includes( "," );
```

Comment:
As far as I know, IE 11 doesn't support includes. I see we use it in the serialization too (will open an issue) but we're also using lodash includes.